view/adminhtml/templates/new_shipment.phtml

```diff
@@ -140,31 +133,24 @@ foreach ($block->getNewShipmentForm()->getCarrierSpecificAbstractConsignments()
                         title="">
                         <?php
                         while ($label_amount <= 10) {
-                            echo '<option value="' . $label_amount . '" >' . __($label_amount) . '</option>';
+                            echo '<option value="', $label_amount, '" >', __($label_amount), '</option>';
                             $label_amount++;
                         }
                         ?>
                     </select>
                 </div>
                 <?php
-                $isInternational = AbstractConsignment::CC_NL !== $block->getCountry();
-
                 try {
                     $abstractConsignment->setPackageType($packageTypeId);
                 } catch (\Exception $e) {
                     continue;
                 }
                 foreach ($abstractConsignment->getAllowedShipmentOptions() as $shipmentOption) {
-                    if (
-                        $isInternational
-                        || AbstractConsignment::SHIPMENT_OPTION_INSURANCE === $shipmentOption
-                        || ! $abstractConsignment->canHaveShipmentOption($shipmentOption)
+                    if (AbstractConsignment::SHIPMENT_OPTION_INSURANCE === $shipmentOption
```

Comment:
Zou je de "insurance" uitzondering niet ook in `consigmentHasShipmentOption` willen hebben, of is er een andere use case waarin dat dan weer niet handig is?

Reply:
Puur vanwege die reden vond ik het duidelijker om hem apart te houden.